cmdstanpy/stanfit.py

```diff
@@ -228,6 +253,12 @@ def _validate_csv_files(self) -> None:
         self._first_draw = dzero.get('first_draw')
         self._metric_type = dzero.get('metric')
 
+    def _assemble_generated_quantities(self) -> None:
+        df_list = []
+        for chain in range(self._chains):
+            df_list.append(pd.read_csv(self.csv_files[chain], comment='#'))
+        self._generated_quantities = pd.concat(df_list).to_numpy()
```

Comment:
Can we use `.values` or `np.array` because `to_numpy` is only in the latest pandas.

Reply:
Also, does this handle int and float correctly?

Reply:
Replaced with values.